samples/drivers/uart/passthrough/sample.yml

```diff
@@ -0,0 +1,13 @@
+sample:
+  name: UART Passthrough
+tests:
+  sample.drivers.uart:
+    integration_platforms:
+      - qemu_x86
+    tags:
+      - serial
+      - uart
+    filter: CONFIG_SERIAL and
+            CONFIG_UART_INTERRUPT_DRIVEN and
+            dt_chosen_enabled("zephyr,shell-uart")
```

Comment:
Shouldn't this be `zephyr,console` instead to mirror `DEV_CONSOLE` below? Also another `dt_chosen_enabled` for `uart,passthrough` might be a good idea.

Reply:
Good spot, thanks!

Reply:
Good spot, thanks... fixed!